[IMP] point_of_sale: add Cashdro cash machine docs (#17119)
odoo/documentation: content/applications/sales/point_of_sale/payment_methods/cash_machines/cashdro.rst

=======
Cashdro
=======

**Cashdro** :doc:`cash machines <../cash_machines>` enable the automation of cash transactions.

.. note::
   - The integration with the Cashdro cash machine only supports payments and refunds.
   - Tasks like filling and emptying the machine must be performed directly through the cash machine
     interface.

.. _pos/cashdro/configuration:

Configuration
=============

Cash machine settings
---------------------

> 

Reviewer: I would add a short introduction here to briefly explain that setting up the cash machine requires locating the IP address (which is going to be used when creating a payment method; so I would add a ref to the **Payment method** section of this page), and enabling HTTP (briefly mention why it's also important).

     the IP address has been noted down.
   - The IP address should be static to ensure long term stable operation.
#. Enable HTTP on the Cashdro machine.

   - Navigate to the login screen at `https://<cashdro-ip>/Cashdro3Web/#/login`.
   - Login as the support user.
   - Navigate to :menuselection:`Configuration --> General parameters`.
   - In the :guilabel:`Web Integration` section, ensure :guilabel:`Enable HTTP` is checked.
   - Restart the machine.

.. note::
   This setup is technical and requires knowing the admin password of the Cashdro hardware. It is advisable to
   consult the Cashdro integration partner for configuration if there is any uncertainty.

Payment method
--------------

#. :ref:`Install <general/install>` the POS Cashdro module.
#. :doc:`Associate a cash payment method <../../payment_methods>`:

   - Go to :menuselection:`Point of Sale --> Configuration --> Payment Methods`. Create a new
     :guilabel:`Cash` payment method or update an existing payment method.
   - Select the associated POS in the :guilabel:`Point of Sale` field.
   - Select :guilabel:`Cash Machine (Cashdro)` in the :guilabel:`Integration` field.

>    - Fill in the :guilabel:`Cashdro IP` field with the IP address of the cash machine.

Reviewer: ```suggestion
   #. Type the Cashdro machine's IP address in the :guilabel:`Cashdro IP` field.
```
